modules/deleting-files-in-trash-directory.adoc

```diff
@@ -0,0 +1,31 @@
+:_module-type: PROCEDURE
+
+[id="deleting-files-in-trash-directory_{context}"]
+= Deleting files into the trash directory of your workbench in Jupyter notebooks
+
+[role='_abstract']
+You can delete unwanted files or folders in the Jupyter Notebook by moving them to your trash directory.
+
+.Prerequisites
+* You have logged in to JupyterLab and opened a Jupyter notebook.
+* You have at least one file or folder that you want to delete.
+
+.Procedure
+. Access the File Browser
+* In your JupyterLab interface, click the *File Browser* (the folder icon) in the left sidebar to see your project files.
+. Select the File
+* Locate the file or folder you wish to delete.
+. Delete the File
+* Right-click on the file or folder's name.
```

Comment:
Especially more as the ordered list here bc its a multiple step ordered operation

suggestion
```
.. Right-click on the file or folder's name.
.. Select *Move to Trash* from the context menu.
.. Confirm the deletion when prompted.
```